Cartfile

```diff
@@ -5,4 +5,4 @@ github "rs/SDWebImage" ~> 4.1
 github "Project-OSRM/osrm-text-instructions.swift" ~> 0.3.1
 github "frederoni/aws-sdk-ios" "ac6a35f135aca73c9d308e960a501fcc16a586a8"
 github "mapbox/mapbox-telemetry-ios" ~> 0.2
-github "https://github.com/bsudekum/Solar" "c162c697652c3dfe389717f6c9070a0421b3f0cd"
+github "https://github.com/1ec5/Solar" "5ce2ca7baaa82b1abb0467642ae237faf674b645"
```

Comment:
This is what I used locally:

```carthage
github "1ec5/Solar" "scheme-test-only"
```

The Cartfile.resolved will end up with a specific hash.